modules/assim.batch/R/pda.define.llik.R

```diff
@@ -0,0 +1,34 @@
+##' Define PDA Likelihood Functions
+##'
+##' @title Define PDA Likelihood Functions
+##' @param all params are the identically named variables in pda.mcmc / pda.emulator
+##'
+##' @return List of likelihood functions, one for each dataset to be assimilated against.
+##'
+##' @author Ryan Kelly
+##' @export
+pda.define.llik.fn <- function(settings) {
```

Comment:
I think we should be passing the model output and observed data explicitly, rather than assuming those exist as global variables. We should also be explicit about passing a parameter vector and the likelihood. If we do that we shouldn't also need to pass the settings object. If we do this we probably want all these things to be lists, with length equivalent to the number of inputs.